modules/italian_strings.py

```diff
@@ -0,0 +1,111 @@
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights 2020                                                       ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+import datetime
+
+
+WORDS_MAP = {
+  'A': 'a',
+  'Dei': 'dei',
+  'Del': 'del',
+  'Di': 'di',
+  'Dott.Ri': 'Dott.ri',
+  'Dott.Ssa': 'Dott.ssa',
+  'Dott.Sse': 'Dott.sse',
+  'Dr.I': 'Dott.ri',
+  'Dr.Ssa': 'Dott.ssa',
+  'Ed': 'ed',
+  'E': 'e',
+  'F.Lli': 'F.lli',
+  'In': 'in',
+  'Responsabilita\'': 'Responsabilità',
+  'Sas': 'S.A.S.',
+  'Sigla': 'sigla',
+  'Snc': 'S.N.C.',
+  'Societa\'': 'Società',
+  'S.P.A.': 'S.p.A.',
+  'Spa': 'S.p.A.',
+  'Srl': 'S.R.L.',
+  'Vii': 'VII',
+  'Vi': 'VI',
+  'Xiv': 'XIV',
+  'Xxiii': 'XXIII',
+  'Xx': 'XX',
+}
+
+# First Char Uppercase
+# quotes (") removal
+# asterisk (*) removal
+# extra spaces trim
+# special case stopwords
+
+
+def normalize(s):
+    s = s.replace('"', ' ').replace('*', ' ').title()
+    s = ' '.join(map(lambda x: WORDS_MAP.get(x, x), s.split()))
+    return s[:1].upper() + s[1:]
+
+
+def osmRefVatin(s):
+    if len(s) != 11 or s.isdigit() is False:
+        return None
+    return 'IT' + s
+
+
+def osmStartDate(s):
+    try:
+        dt = datetime.datetime.strptime(s, '%d/%m/%Y')
+        return dt.date().__str__()
+    except ValueError:
+        return None
```

Comment:
It is not an Italian specific.

Reply:
Right... should I move it somewhere else?

Reply:
Maybe here, there is already fee tools: https://github.com/osm-fr/osmose-backend/blob/master/analysers/Analyser_Merge.py#L890